includes/commands/class-mu-migration-users.php

```diff
@@ -88,11 +89,10 @@ public function update_passwords( $args = array(), $assoc_args = array() ) {
 	}
 
 	/**
-	 * Based on retrieve_passwords
+	 * Handles sending password retrieval email to user (based on retrieve_password).
 	 *
 	 * @param $user_data
```

Comment:
This should actually be of type `WP_User` (which then gets passed to `get_password_reset_key`), but it's being passed as an `object`.

It seems to work anyway, but for IDEs it's nicer to have the proper parameter annotated.

As I'm not 100% sure if this has any side-effects (using `WP_User` instead of the `WP_User->data` object), I left it.